src/components/TodoListRender.js

```diff
@@ -0,0 +1,21 @@
+const TodoListRender = ({$todoList, currentState, currentFilterState}) => {
+    if(currentFilterState === "all") {
+        currentState = currentState
+    } else if(currentFilterState === "active") {
+        currentState = currentState.filter(todo => todo.isCompleted === false)
+    } else {
+        currentState = currentState.filter(todo => todo.isCompleted === true)
+    }
+    $todoList.innerHTML = currentState.map(
+        (todo, index) => `<li data-index="${index}" class="${todo.isCompleted ? `completed` : `false`}">
```

Comment:
```suggestion
        (todo, index) => `<li data-index="${index}" class="${todo.isCompleted && `completed`}">
```

false라는 classname에 마땅한 의미가 없다면 isCompleted일 때만 classname을 주입하는 방식으로도 사용할 수 있어요.

Reply:
네! 맞아요 `false` 는 다른 특별한 의미가 내포되지는 않습니다.
그럼 말씀하신대로 수정해보겠습니당.